### PR TITLE
proofread: destroy.md

### PR DESCRIPTION
**Pull request by rayw000**
link to: https://github.com/juicedata/docs/issues/124

Hi @AIXjing , could you please review this PR when you get time? Thank you!

### CODE REVIEW

File: docs/en/administration/destroy.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ position: 7
 ---
 # How to destroy a file system
 
-JuiceFS client provides the `destroy` command to completely destroy a file system, which will result in the following.
+JuiceFS client provides a `destroy` command to completely destroy a file system, which will result in the following.
```
Comment (AIXjing) [inline docs/en/administration/destroy.md:7]:
```suggestion
JuiceFS client provides the `destroy` command to completely destroy a file system, which will give the following results.
```

File: docs/en/administration/destroy.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ position: 7
...
 
-- Delete all metadata entries of this file system
+- Delete all metadata entries of the file system
```
Comment (AIXjing) [inline docs/en/administration/destroy.md:9]:
```suggestion
- Deletion of all metadata entries of this file system
```
这里用不用强调是「此」文件系统

File: docs/en/administration/destroy.md [hunk 3]
```diff
@@ -4,12 +4,12 @@ position: 7
...
 - Deletes all data blocks of the file system
```
Comment (AIXjing) [inline docs/en/administration/destroy.md:10]:
```suggestion
- Deletion of all data blocks of this file system
```

File: docs/en/administration/destroy.md [hunk 4]
```diff
@@ -4,12 +4,12 @@ position: 7
...
 
-The command to destroy a file system is as follows.
+Use this command in the following format.
```
Comment (AIXjing) [inline docs/en/administration/destroy.md:12]: 改之后的好像有些不清楚，感觉原来的好一些？

File: docs/en/administration/destroy.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ $ juicefs status redis://127.0.0.1:6379/1
 ## Destroy a file system
 
 :::danger
-The destroy operation will cause all the data in the database records and object storage associated with the file system to be deleted, please make sure to backup the important data first before operation!
+The destroy operation will cause all the data in the database and the object storage associated with the file system to be deleted, please make sure to backup the important data first before operating!
```
Comment (AIXjing) [inline docs/en/administration/destroy.md:52]:
```suggestion
The destroy operation will cause all the data in the database and the object storage associated with the file system to be deleted. Please make sure to back up the important data before operating!
```

File: docs/en/administration/destroy.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ $ juicefs status redis://127.0.0.1:6379/1
 ## Destroy a file system
 
 :::danger
-The destroy operation will cause all the data in the database records and object storage associated with the file system to be deleted, please make sure to backup the important data first before operation!
+The destroy operation will cause all the data in the database and the object storage associated with the file system to be deleted. Please make sure to back up the important data before operating!
```
Comment (timfeirg) [inline docs/en/administration/destroy.md:52]: remove the "please"

File: docs/en/administration/destroy.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ juicefs destroy <METADATA URL> <UUID>
 
 ## Find the UUID of the file system
 
-The `status` command on the JuiceFS client can view detailed information about a file system by simply specifying the file system's metadata engine URL, e.g.
+JuiceFS client provides a `status` command to view detailed information about a file system by simply specifying the file system's metadata engine URL, e.g.
```
Comment (timfeirg) [inline docs/en/administration/destroy.md:23]: Use `juicefs status` to check detailed information about a file system, e.g.

File: docs/en/administration/destroy.md [hunk 8]
```diff
@@ -4,12 +4,12 @@ position: 7
 ---
 # How to destroy a file system
 
-JuiceFS client provides the `destroy` command to completely destroy a file system, which will result in the following.
+JuiceFS client provides the `destroy` command to completely destroy a file system, which will result in
```
Comment (timfeirg) [inline docs/en/administration/destroy.md:7]: need trailing colon?

**Review by timfeirg — APPROVED** [review]

File: docs/en/administration/destroy.md (hunk 4 above)
Comment (timfeirg) [reply docs/en/administration/destroy.md:12]: just use "command usage"

**Review by timfeirg — APPROVED** [review]

File: docs/en/administration/destroy.md (hunk 4 above)
Comment (rayw000) [reply docs/en/administration/destroy.md:12]: `Command Usage`, or just a `Usage` is better to me than doing literal translation.

**Pull request merged**